_articles/ms/building-community.md

```diff
@@ -0,0 +1,280 @@
+
+* Sekiranya projek anda berada di GitHub, **pindahkan projek anda dari akaun peribadi anda ke [Organization](https://help.github.com/articles/creating-a-new-organization-account/)** dan tambahkan sekurang-kurangnya satu pentadbir sandaran. Organisasi menjadikannya lebih mudah untuk mengerjakan projek dengan kolaborator luar.
+
+Kenyataannya adalah bahawa[most projects only have](https://peerj.com/preprints/1233.pdf)satu atau dua penyelenggara yang melakukan sebahagian besar kerja. Semakin besar projek anda, dan semakin besar komuniti anda, semakin mudah mendapatkan bantuan.
+
+Walaupun anda mungkin tidak selalu menjumpai seseorang untuk menjawab panggilan tersebut, memberi isyarat di luar sana akan meningkatkan kemungkinan orang lain akan masuk. Dan lebih awal anda memulakannya, semakin cepat orang dapat membantu.
+
+<aside markdown="1" class="pquote">
+  <img src="https://avatars.githubusercontent.com/gr2m?s=180" class="pquote-avatar" alt="avatar">
+  \[Ia ada di dalam anda\] minat terbaik untuk merekrut penyumbang yang menikmati dan yang mampu melakukan perkara yang bukan anda lakukan. Adakah anda menikmati pengekodan, tetapi tidak menjawab masalah? Kemudian kenal pasti individu-individu dalam komuniti anda yang melakukannya dan biarkan mereka memilikinya.
+  <p markdown="1" class="pquote-credit">
+— @gr2m, ["Welcoming Communities"](http://hood.ie/blog/welcoming-communities.html)
+  </p>
+</aside>
+
+## Menyelesaikan konflik
+
+Pada peringkat awal projek anda, membuat keputusan besar adalah mudah. Apabila anda ingin melakukan sesuatu, anda hanya melakukannya.
+
+Oleh kerana projek anda menjadi lebih popular, lebih ramai orang akan tertarik dengan keputusan yang anda buat. Walaupun anda tidak mempunyai komuniti penyumbang yang besar, jika projek anda mempunyai banyak pengguna, anda akan mendapati orang mempertimbangkan keputusan atau membangkitkan masalah mereka sendiri.
+
+Sebahagian besarnya, jika anda telah membina komuniti yang ramah, hormat dan mendokumentasikan proses anda secara terbuka, komuniti anda seharusnya dapat mencari penyelesaian. Tetapi kadang-kadang anda menghadapi masalah yang agak sukar untuk ditangani.
+
+### Tetapkan bar untuk kebaikan
+
+Apabila komuniti anda bergelut dengan masalah yang sukar, kemarahan mungkin akan meningkat. Orang mungkin menjadi marah atau kecewa dan mengeluarkannya satu sama lain, atau pada anda.
+
+Tugas anda sebagai penyelenggara adalah untuk memastikan situasi ini tidak meningkat. Walaupun anda mempunyai pendapat yang kuat mengenai topik ini, cubalah mengambil kedudukan sebagai moderator atau fasilitator, daripada melompat ke dalam pertengkaran dan mendorong pandangan anda. Sekiranya seseorang bersikap tidak baik atau memonopoli perbualan, [act immediately](../building-community/#dont-tolerate-bad-actors) agar perbincangan tetap sopan dan produktif.
```

Comment:
@KaynenStrife the fragment identifier `#dont-tolerate-bad-actors` will be different because the headings (which generate fragment identifiers are translated).

Take a look at the fragment identifiers in another translation, eg the same file in zh-hant, for how to do it.

Reply:
(The id translation may be another good example to look at -- translated headings, and you can see that the fragment identifiers are lowercased, with `-` substituting for spaces.)

Reply:
Took me a while to set the internal links right, I hope this fulfills the conditions required for a merge to occur.

Also, thank you very much @mlinksva for pointing me in the right direction. Would have used up a full day just to figure out where my error came from.